src/vs/workbench/parts/extensions/browser/extensionsActions.ts

```diff
@@ -1717,6 +1788,149 @@ export class DisabledStatusLabelAction extends Action {
 	}
 }
 
+// TODO: test when workspace is folder
+//				TODO: test when recomendation file already exists
+//							TODO: test when current extension is already recommended
+//							TODO: test when current extension is NOT recommended
+//				TODO: test when recomendation file doesnt exists
+// TODO: test when workspace is workspace
+//				TODO: test when recomendation file already exists
+//							TODO: test when current extension is already recommended
+//							TODO: test when current extension is NOT recommended
+//				TODO: test when recomendation file doesnt exists
+// TODO: test when workspace is empty
+// TODO: make sure we preserve comments
+
+export class RecommendToFolderAction extends AbstractConfigureRecommendedExtensionsAction {
+	private static readonly Class = 'extension-action recommend-to-folder';
```

Comment:
Css wise the 2 actions look the same. Then why not use the same css class for both?